redisgraph/query_result.py

```diff
@@ -46,6 +46,7 @@ def _retrieve_data_from_statistics(self, statistics):
     @staticmethod
     def _get_value(prop, statistics):
         for stat in statistics:
+            stat = stat.decode()
```

Comment:
I'm not that familiar with Python3, could you explain why do we need decode here?